openedx/features/content_type_gating/partitions.py

```diff
@@ -82,7 +82,8 @@ def access_denied_fragment(self, block, user, user_group, allowed_groups):
         course_key = self._get_course_key_from_course_block(block)
         modes = CourseMode.modes_for_course_dict(course_key)
         verified_mode = modes.get(CourseMode.VERIFIED)
-        if verified_mode is None or not self._is_audit_enrollment(user, block):
+        if (verified_mode is None or not self._is_audit_enrollment(user, course_key) or
+                user_group.name == FULL_ACCESS.name):
```

Comment:
Nit: I think groups compare correctly, so you can probably just use `user_group == FULL_ACCESS`